howto/descriptor.po

```diff
@@ -50,17 +51,25 @@ msgid ""
 "methods.  Shows how each works by giving a pure Python equivalent and a "
 "sample application."
 msgstr ""
+"Définit les descripteurs, résume le protocole et montre comment les "
```

Comment:
Je ne sais pas si l'on est obligé de coller au style télégraphique de Raymond. There must be a better way ! et je tape du poing sur la table.
Bref, je pense que l'on peut construire de vraies phrases